dashboard/checkit/outcome.py

```diff
@@ -96,14 +96,14 @@ def build_path(self):
     def seeds_json_path(self):
         return os.path.join(self.build_path(),"seeds.json")
 
-    def generate_exercises(self,regenerate=False,images=False):
+    def generate_exercises(self,amount,regenerate=False,images=False):
```

Comment:
let's make amount a keyword argument with a default of 1000 just in case this method is called somewhere else, e.g. `amount=1_000` instead of just `amount`